stories/core/RatingBar.js

```diff
@@ -2,6 +2,14 @@ import React from 'react'
 import { RatingBar } from 'react-ui-core/src'
 import theme from '../theme/core/RatingBar.css'
 
+const handleRatingClick = ({ pageX, currentTarget }) => {
+  const getBoundingClientRect = currentTarget.getBoundingClientRect()
+  const starValue =
+    Math.ceil(((pageX - getBoundingClientRect.x) / getBoundingClientRect.width) * 5)
+  // eslint-disable-next-line no-alert
+  window.alert(starValue)
```

Comment:
Rather than an alert, in the react-ui stories you can do something like this:
```
import { action } from '@storybook/addon-actions'
action(`Clicked star value`)(starValue)
```